scanner/k8s-assets/processor/processor.go

```diff
@@ -317,6 +317,28 @@ func (p *Processor) ProcessContainer(
 		componentVersionId  string
 		componentInstanceId string
 	)
+
+	// Is this a sidecar mapped to a specific service?
+	advCfg, err := p.config.LoadAdvancedConfig()
```

Comment:
Is it possible to load that config only once? Otherwise it might spam the logs if it can't be read for any container